test/unit/MerkleDrop.spec.js

```diff
@@ -247,10 +372,38 @@ describe(">> MerkleDrop", () => {
         );
       });
     });
+
+    describe("with non-existent tranche", () => {
+      let proof, expectedBalance;
+
+      const initialState = {
+        ...commonState,
+        withProof: true,
+        zeroAllocation: true,
+      };
+      const nonExistentTrancheIdx = "5";
+
+      it("reverts 'Week cannot be in the future'", async () => {
```

Comment:
Just interesting, why exactly `Week cannot be in the future` for nonexistent tranche?

Reply:
Yea, I put off the brain and just copied what was already in the MerkleDrop here. I just changed the error message now. thanks